testdata/public/package/default-pipeline-0.0.2/index.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "default-pipeline",
+  "title": "Default pipeline Integration",
+  "version": "0.0.2",
+  "description": "Tests if no pipeline is set, it defaults to the default one",
+  "type": "integration",
+  "categories": [
+    "logs"
+  ],
+  "requirement": {
+    "kibana": {}
+  },
+  "assets": [
+    "/package/default-pipeline-0.0.2/manifest.yml",
+    "/package/default-pipeline-0.0.2/dataset/foo/manifest.yml",
+    "/package/default-pipeline-0.0.2/dataset/foo/elasticsearch/ingest-pipeline/default.json"
+  ],
+  "format_version": "1.0.0",
+  "datasets": [
+    {
+      "title": "Foo",
+      "name": "foo",
+      "release": "beta",
+      "type": "logs",
+      "ingest_pipeline": "default"
```

Comment:
@skh This PR made me think: Should I return here `default.json` or `default.yml`. This work could be done automatically by the registry for you. It is only in the manifest where we simplify things for the user.